static/app/components/arithmeticBuilder/token/index.spec.tsx

```diff
@@ -518,6 +531,38 @@ describe('token', function () {
       ).toBeInTheDocument();
     });
 
+    it('maps key to suggested key on enter', async function () {
+      render(<Tokens expression="avg(span.duration)" />);
+
```

Comment:
You should also have an integration where the ui component gets used and mock out a trace-item-attributes response